### PR TITLE
Nullsafety support

### PR DESCRIPTION
**Pull request by sergioelisan**

### CODE REVIEW

**Comment by Ne4istb** [comment]
Thanks for your PR. It will be reviewed soon.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: verbal_expressions
 description: A package that helps to construct difficult regular expressions.
-version: 0.4.0
-author: Ne4istb <anechytailov@gmail.com>
+version: 1.0.0-nullsafety
+author: Ne4istb <anechytailov@gmail.com>, sergiolisan <sergiolisan@gmail.com>
```
Comment (Ne4istb) [inline pubspec.yaml]:
```suggestion
version: 0.5.0-nullsafety
author: Ne4istb <anechytailov@gmail.com>
contributors: sergiolisan <sergiolisan@gmail.com>
```

File: lib/src/verbal_expression.dart [hunk 2]
```diff
@@ -1,15 +1,30 @@
+///
+///
+///
```
Comment (Ne4istb) [inline lib/src/verbal_expression.dart]:
```suggestion
```

File: lib/src/range.dart [hunk 3]
```diff
@@ -1,19 +1,26 @@
+///
+///
+///
```
Comment (Ne4istb) [inline lib/src/range.dart]:
```suggestion
```

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Ne4istb) [inline .vscode/launch.json]: please, exclude this file and .DS_Store files

**Review by Ne4istb — CHANGES_REQUESTED** [review]
Thank you for your contribution. 
Please, fix minor issues mentioned in the comments and this PR will be merged to master branch.

**Comment by Ne4istb** [comment]
And please resolve merge conflict

**Review by Ne4istb — CHANGES_REQUESTED** [review]
Please, delete 'test/.DS_Store'

**Comment by sergioelisan** [comment]
I'm sorry, those .DS_Store are a plague!

**Pull request merged**